puni.el

```diff
@@ -2208,6 +2240,24 @@ With positive prefix argument N, barf that many sexps."
      puni-blink-region-face)
     (setq deactivate-mark nil)))
 
+;;;###autoload
+(defun puni-splice-killing-backward ()
+  "Splice the list around point by removing its delimiters, and
```

Comment:
The style checker is complaining on this one. What about

```
  "Kill all sexps before point in the current list, then splice it.
Splicing is done by removing the delimiters of the list."
```

The same to `puni-splice-killing-backward`.

Reply:
done